Lib/lib2to3/fixes/fix_urllib.py

```diff
@@ -13,15 +13,15 @@
                 ("urllib.request",
                     ["URLopener", "FancyURLopener", "urlretrieve",
                      "_urlopener", "urlopen", "urlcleanup",
-                     "pathname2url", "url2pathname"]),
+                     "pathname2url", "url2pathname", "getproxies"]),
                 ("urllib.parse",
                     ["quote", "quote_plus", "unquote", "unquote_plus",
                      "urlencode", "splitattr", "splithost", "splitnport",
                      "splitpasswd", "splitport", "splitquery", "splittag",
                      "splittype", "splituser", "splitvalue", ]),
                 ("urllib.error",
                     ["ContentTooShortError"])],
-           "urllib2" : [
+           "urllib2": [
```

Comment:
Please also revert this whitespace change.

Reply:
@tirkarthi Done!